display/display.go

```diff
@@ -245,43 +245,36 @@ func Init() {
 			return event
 		}
 
+		/* To add a configurable global key command, you'll need to update one of
+		   the two switch statements here.  You'll also need to add an enum entry in
+		   config/keybindings.go, update KeyInit() in config/keybindings.go, add a default
+		   keybinding in config/config.go and update the help panel in display/help.go
+		*/
```

Comment:
```suggestion
		// To add a configurable global key command, you'll need to update one of
		// the two switch statements here.  You'll also need to add an enum entry in
		// config/keybindings.go, update KeyInit() in config/keybindings.go, add a default
		// keybinding in config/config.go and update the help panel in display/help.go
		
```
Go code tends to avoid the block style comments.